src/components/TokenBalance.tsx

```diff
@@ -233,7 +217,7 @@ export function AssetsTokenBalance({
         <TokenBalance
           style={styles.totalBalance}
           singleTokenViewEnabled={false}
-          showBalanceToggle={isWalletTab}
+          showBalanceToggle={true}
```

Comment:
I believe this prop can now be removed from TokenBalance

Reply:
FiatExchangeTokenBalance also uses this but that doesn't need it. I think we should just make them different components, but we may change our withdraw spend page, so we can hold off for now